__tests__/integration_testing.test.js

```diff
@@ -129,4 +129,41 @@ describe('getComments', () => {
             });
         })
     }); 
+});
+
+describe('postComments', () => {
+    it('should return a 201 status code', () => {
+        const comment = {username: 'butter_bridge', body: 'a'}
+        return request(app).post(`/api/articles/1/comments`).send(comment).expect(201)
+    });
+    it('should return the posted comment with the username and body properties', () => {
+        const comment = {username: 'butter_bridge', body: 'a'}
+        const expected = {username: 'butter_bridge', body: 'a'}
+        return request(app).post(`/api/articles/1/comments`).send(comment).expect(201).then((res) => {
+            expect(comment).toEqual(expected)
+            expect(comment).toHaveProperty('username')
+            expect(comment).toHaveProperty('body')
+        })
+    });
+    it('should not return the created_at, votes, comment_id or article_id properties', () => {
+        const comment = {username: 'butter_bridge', body: 'a'}
+        return request(app).post(`/api/articles/1/comments`).send(comment).expect(201).then((res) => {
+            expect(comment.created_at).toBeUndefined()
+            expect(comment.votes).toBeUndefined()
+            expect(comment.comment_id).toBeUndefined()
+            expect(comment.article_id).toBeUndefined()
+        })
+    });
+    it('should return a 400 status code if bad request', () => {
```

Comment:
should return a 400 when? Ensure we are descriptive of the behaviour we are testing for in the test description.